client/state/plans/assembler.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+export const createPlanObject = plan => {
+	return {
+		androidSku: String( plan.android_sku ),
+		appleSku: String( plan.apple_sku ),
+		available: String( plan.available ),
+		billPeriod: Number( plan.bill_period ),
+		billPeriodLabel: String( plan.bill_period_label ),
+		cost: Number( plan.cost ),
+		capability: String( plan.capability ),
```

Comment:
Be careful when converting! JS is not the greatest at converting from one type to another. Do we really need all these type conversions?
<img width="244" alt="screen shot 2016-05-09 at 12 41 51 pm" src="https://cloud.githubusercontent.com/assets/1270189/15125811/a124a126-15e3-11e6-84ac-325a4b893422.png">